asmcomp/arm64/emit.mlp

```diff
@@ -327,12 +332,16 @@ let float_literal env fl =
 let emit_literals env =
   if env.float_literals <> [] then begin
     if macosx then
-      `	.section	__TEXT,__literal8,8byte_literals\n`;
+      `	.section	__TEXT,__literal8,8byte_literals\n`
+    else if not text_section_is_readable then
+      `	.section .rodata\n`;
```

Comment:
I'm in favor of putting literals in read-only data sections for all ports, incl. Linux.  It should improve code cache utilization a bit.